2020Labs/RHELSecurity/ansible/udica/udica.yml

```diff
@@ -0,0 +1,29 @@
+---
+- hosts: all
+  become: yes
+
+  tasks:
+
+  - name: Add the user 'user'
+    user:
+      name: user
+
+  - name: Set authorized key taken from file for root
+    authorized_key:
+      user: root
+      state: present
+      key: "{{ lookup('file', 'id_rsa.pub') }}"
+
+  - name: Set authorized key taken from file for adminlab
+    authorized_key:
+      user: auditlab
+      state: present
+      key: "{{ lookup('file', 'id_rsa.pub') }}"
```

Comment:
Drop both authorized_key's - for root and for user which is later auditlab

Reply:
@wrabcak : can you update the udica.yml to do what @bachradsusi suggested above and submit a new PR? Also, I noticed that this PR has a conflict we need to resolve. Can you look at that too?